sycl/include/CL/sycl/detail/builtins.hpp

```diff
@@ -58,11 +73,12 @@
     using Arg1 = cl::sycl::detail::ConvertToOpenCLType_t<T1>;                  \
     using Arg2 = cl::sycl::detail::ConvertToOpenCLType_t<T2>;                  \
     using Arg3 = cl::sycl::detail::ConvertToOpenCLType_t<T3>;                  \
-    extern Ret PPCAT(prefix, call)(Arg1, Arg2, Arg3);                          \
+    EXTERN_IT3(Ret, prefix, call, Arg1, Arg2, Arg3);                           \
     Arg1 arg1 = cl::sycl::detail::convertDataToType<T1, Arg1>(t1);             \
     Arg2 arg2 = cl::sycl::detail::convertDataToType<T2, Arg2>(t2);             \
     Arg3 arg3 = cl::sycl::detail::convertDataToType<T3, Arg3>(t3);             \
-    Ret  ret  = PPCAT(prefix, call)(arg1, arg2, arg3);                         \
+    Ret  ret  = PPCAT(prefix, call)TPARAMS3(Ret, Arg1, Arg2, Arg3)(            \
```

Comment:
```suggestion
    Ret ret  = PPCAT(prefix, call)TPARAMS3(Ret, Arg1, Arg2, Arg3)(             \
```

Reply:
This codestyle is already there. I won't like to change it.

Reply:
Actually, this code style is not respected for declaration of `Ret` type. Personally, I don't see any problems with changing it

Reply:
Fine.